src/spaces/index.js

```diff
@@ -1,22 +1,9 @@
-import "./xyz-d65.js";
-import "./xyz-d50.js";
-import "./xyz-abs-d65.js";
-import "./lab.js";
-import "./lch.js";
-import "./srgb-linear.js";
-import "./srgb.js";
-import "./hsl.js";
-import "./hwb.js";
-import "./hsv.js";
-import "./p3-linear.js";
-import "./p3.js";
-import "./a98rgb-linear.js";
-import "./a98rgb.js";
-import "./prophoto-linear.js";
-import "./prophoto.js";
-import "./rec2020-linear.js";
-import "./rec2020.js";
-import "./oklab.js";
-import "./oklch.js";
+import ColorSpace from "../space.js";
 
-import "./index-hdr.js";
\ No newline at end of file
+import * as spaces from './index-fn.js';
+
+for (let key of Object.keys(spaces)) {
+	ColorSpace.register(spaces[key]);
+}
+
+// ...
```

Comment:
Don't forget to re-export!

Reply:
Oh, do we want to re-export from this file as well? No problem, can do 🙂